src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -58,6 +58,8 @@ public class TestConnectionCommand : PSCmdlet, IDisposable
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.Ping.returns false without errors for an unresolvable address when using -Quiet
```powershell
Expected $false, but got $true.
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 65
65:             $result | Should -BeFalse
```